CHEWBBACA/SchemaEvaluator/schema_evaluator.py

```diff
@@ -0,0 +1,703 @@
+import os
+import sys
+import json
+import statistics
+import pandas as pd
+import multiprocessing
+from operator import itemgetter
+from collections import Counter
+
+
+from Bio import SeqIO
+from Bio.Seq import Seq
+from Bio.SeqRecord import SeqRecord
+from Bio.Align.Applications import MafftCommandline
+from Bio.Align.Applications import ClustalwCommandline
+
+try:
+    from utils import auxiliary_functions as aux
+except:
+    from CHEWBBACA.utils import auxiliary_functions as aux
+
+
+# Schema Evaluator Auxiliary Functions
+
+def gene_seqs_info_schema_evaluator(gene):
+    """ Determines the total number of alleles and the mean length
```

Comment:
To be consistent with the documentation in other modules that have been added or refactored, docstrings for new modules and functions should follow the [NumPy](https://numpydoc.readthedocs.io/en/latest/format.html). This should also help when we create new documentation in Read the Docs.